From 91835a81d554fb5756f919750553936f28656e39 Mon Sep 17 00:00:00 2001
From: Dmitri Smirnov <dmixsmi@gmail.com>
Date: Thu, 21 Dec 2023 10:42:04 -0500
Subject: [PATCH] fix: add missing support for OptiX 7.6

---
 CSGOptiX/OPT.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CSGOptiX/OPT.h b/CSGOptiX/OPT.h
index 077bb3f22..c6175955b 100644
--- a/CSGOptiX/OPT.h
+++ b/CSGOptiX/OPT.h
@@ -12,7 +12,7 @@ struct OPT
     static constexpr const char* OPTIX_COMPILE_DEBUG_LEVEL_NONE_     = "OPTIX_COMPILE_DEBUG_LEVEL_NONE" ; 
     static constexpr const char* OPTIX_COMPILE_DEBUG_LEVEL_LINEINFO_ = "OPTIX_COMPILE_DEBUG_LEVEL_LINEINFO" ; 
     static constexpr const char* OPTIX_COMPILE_DEBUG_LEVEL_FULL_     = "OPTIX_COMPILE_DEBUG_LEVEL_FULL" ; 
-#elif OPTIX_VERSION == 70500
+#elif OPTIX_VERSION == 70500 || OPTIX_VERSION == 70600
     static constexpr const char* OPTIX_COMPILE_DEBUG_LEVEL_DEFAULT_  = "OPTIX_COMPILE_DEBUG_LEVEL_DEFAULT" ; 
     static constexpr const char* OPTIX_COMPILE_DEBUG_LEVEL_NONE_     = "OPTIX_COMPILE_DEBUG_LEVEL_NONE" ; 
     static constexpr const char* OPTIX_COMPILE_DEBUG_LEVEL_MINIMAL_  = "OPTIX_COMPILE_DEBUG_LEVEL_MINIMAL" ; 
-- 
2.42.1

