From 3c2a0ac17c2fb6d00cb08c2e7add4660642e8c46 Mon Sep 17 00:00:00 2001
From: Dmitri Smirnov <dmixsmi@gmail.com>
Date: Fri, 24 May 2024 12:41:09 -0400
Subject: [PATCH] fix: update array dtype for numpy >= 1.26

---
 sysrap/sevt.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/sysrap/sevt.py b/sysrap/sevt.py
index 191926c34..a67de5d7e 100644
--- a/sysrap/sevt.py
+++ b/sysrap/sevt.py
@@ -380,13 +380,13 @@ class SEvt(object):
             qtab = eval(qtab_)
             qtab_ = qtab_.replace("q","%s.q" % symbol)
 
-            nosc = np.ones(len(qq), dtype=np.bool )       # start all true
+            nosc = np.ones(len(qq), dtype=bool )       # start all true
             nosc[np.where(qq == pcf.SC)[0]] = 0     # knock out photons with scatter in their histories
 
-            nore = np.ones(len(qq), dtype=np.bool )       # start all true
+            nore = np.ones(len(qq), dtype=bool )       # start all true
             nore[np.where(qq == pcf.RE)[0]] = 0     # knock out photons with reemission in their histories
 
-            noscab = np.ones(len(qq), dtype=np.bool )     # start all true  
+            noscab = np.ones(len(qq), dtype=bool )     # start all true
             noscab[np.where(qq == pcf.SC)[0]] = 0   # knock out photons with scatter in their histories
             noscab[np.where(qq == pcf.AB)[0]] = 0   # knock out photons with bulk absorb  in their histories
         else:
-- 
2.39.3

